test/onnx/verify/matmulinteger_uns_zp_test.cpp

```diff
@@ -43,6 +43,6 @@ TEST_CASE(matmulinteger_uns_zp_test)
     auto result = p.eval(pp).back();
     std::vector<int32_t> result_vector;
     result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
-    std::vector<int32_t> gold = {-38, -83, -44, -98, -50, -113, -56, -128};
+    std::vector<int32_t> gold = {13, 76, 10, 64, 7, 52, 4, 40};
```

Comment:
same question.  Looks like the previous gold value comes from the Onnx doc at https://github.com/onnx/onnx/blob/main/docs/Operators.md#examples-87 .  Why does the answer change?

Reply:
I wrote my own by hand to validate this output